src/Path/AppointmentBooking/AppointmentsUserCreated.jsx

```diff
@@ -0,0 +1,312 @@
+import React, { useState, useEffect, useCallback } from "react";
+import SideBar from "../../Components/SideBar";
+import { useNavigate } from "react-router-dom";
+import { useTranslation } from "react-i18next";
+import "../../index.css";
+import { getAppointmentsUserCreated, deleteAppointments } from "../../Services/appointmentBookingApi";
+import { getListings } from "../../Services/listingsApi";
+import APPOINTMENTDEFAULTIMAGE from "../../assets/Appointments.png";
+
+const AppointmentsUserCreated = () => {
+    const { t } = useTranslation();
+    const [appointments, setMyAppointments] = useState([]);
+    const [pageNumber, setPageNumber] = useState(1);
+    const pageSize = 9;
+
+    const fetchMyServices = useCallback(() => {
+        getAppointmentsUserCreated({
+            pageNumber,
+            pageSize,
+        })
+            .then((response) => {
+                const appointments = response.data.data;
+
+                getListings().then((listingsResponse) => {
+                    const listings = listingsResponse.data.data;
```

Comment:
Bro, you are getting all the listings and then filtering in the frontend you can make a request to the backend with the appointment Id and not do filtering in the frontend.